tests/common.py

```diff
@@ -72,6 +73,17 @@ def sandboxed(*args, **kwargs):
     return path
 
 
+# Context manager for running a test in a directory, e.g. path is the sandbox
+@contextmanager
+def run_in_directory(path):
```

Comment:
Or this could be a decorator?

Reply:
Changed in  https://github.com/PyAV-Org/PyAV/pull/718/commits/90a3479a7d57676e8f91b24b50803536097fdacc